translation/source/site.xml

```diff
@@ -808,4 +808,10 @@ in %3$s</string>
   <string name="opening">Opening</string>
   <string name="middlegame">Middlegame</string>
   <string name="endgame">Endgame</string>
+  <string name="registerToSendChallenges">Please register to send challenges.</string>
+  <string name="youCannotChallengeX">You cannot challenge %s.</string>
+  <string name="xDoesNotAcceptChallenges">%s does not accept challenges.</string>
+  <string name="yourXRatingIsTooFarFromY">Your %s rating is too far from %s.</string>
```

Comment:
this should use explicit ordering of argument, with the `%1$s` syntax, like on https://github.com/ornicar/lila/blob/870af39656d0de8f5fc6acefd0ef5ab1d170571c/translation/source/site.xml#L299

Reply:
good catch! also added to #3453 or maybe trans-dump can lint that